src/index.ts

```diff
@@ -8,7 +8,11 @@ import type { Deployment } from './riffraff';
 import { manifest, riffraffPrefix } from './riffraff';
 import { S3Store, sync } from './s3';
 
-export const main = async (): Promise<void> => {
+interface Options {
+	WithSummary: boolean; // Use to disable summary when running locally.
```

Comment:
Is this simply to de-clutter test output?

Reply:
Ah ... it's to stop the summary appearing in GitHub checks where it doesn't make sense. 👍

Reply:
The test was actually broken (when run locally) by https://github.com/guardian/actions-riff-raff/pull/20 but went unnoticed. The summary requires a GITHUB env var + file to be present, which is hard to stub locally.